src/main/java/uk/gov/hmcts/befta/data/RecentExecutionsInfo.java

```diff
@@ -0,0 +1,44 @@
+package uk.gov.hmcts.befta.data;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class RecentExecutionsInfo {
+
+    public static final String LAST_EXECUTION_TIME = "lastExecutionTime";
+    public static final String LAST_EXECUTION_PROJECT_REPO = "lastExecutionProjectRepo";
+    public static final String LAST_EXECUTION_PROJECT_BRANCH = "lastExecutionProjectBranch";
+
+
+    @JsonProperty(LAST_EXECUTION_TIME)
+    private String lastExecutionTime;
+
+    @JsonProperty(LAST_EXECUTION_PROJECT_REPO)
```

Comment:
Why use the capitalization? We could just use the original class attribute names.